static/app/views/issueDetails/groupActivityItem.tsx

```diff
@@ -96,8 +100,11 @@ function GroupActivityItem({
   organization,
   projectId,
   author,
+  group,
 }: GroupActivityItemProps) {
   const issuesLink = `/organizations/${organization.slug}/issues/`;
+  const isFeedback = (group.issueCategory as string) === 'feedback';
+  const issueType = isFeedback ? t('feedback') : t('issue');
 
   function getIgnoredMessage(data: GroupActivitySetIgnored['data']) {
     const archived = t('archived');
```

Comment:
This is old code, so we don't need to change anything but...

I don't think we should be treating sentences like backtick-concatenated strings. I'd stive to inline words like this as often as possible at the expense of have some more sentences copy+pasted (for each condition) into the app

Reply:
good frontend tsc topic!

Reply:
ya! good idea & callout